### PR TITLE
fix(storybook): Fix fxa settings storybooks

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

- Some storybooks were broken and impacted cross-functional collab

## This pull request

- Fixes storybooks for:
  - images
  - BrandMessaging
  - ConnectedServices
  - Signin
  - SigninRecoveryCode
  - Signup
- Remove ThirdPartyAuthCallback story - the component is not currently used in React and was causing an inception storybook (storybook in storybook) that was taking too long to troubleshoot

## Issue that this pull request solves

Closes: #FXA-10085

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Storybooks for ResetPassword/AccountRecoveryConfirmKey and ResetPassword/AccountRecoveryResetPassword are still broken and not included in this patch.

### CODE REVIEW

File: packages/fxa-settings/src/react-app-env.d.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ declare module '*.webp' {
 }
 
 declare module '*.svg' {
-  import * as React from 'react';
+  import React from 'react';
```
Comment (LZoog) [inline packages/fxa-settings/src/react-app-env.d.ts:58]: Ohh, was this the culprit of some of our broken SVGs?

**Review by LZoog — APPROVED** [review]
LGTM, thank you so much for fixing these! I clicked around and think you got them all? At least the most important ones. :)

(don't forget to squash your commits! 🤪)

**Comment by LZoog** [comment]
On closer inspection we probably want a follow up issue filed to look at our signin stories at least:
* Shouldn’t pocket have other links showing? 
* Should we have a similar state^ for Monitor?
*  “Sign in to pocket with cached credentials” and “with password” are the same. If it's intentional because Pocket always needs the password, fine, but it's not super clear
* Not sure if we want to tweak the "No Password" in "No Linked Account And No Password wording", that sounds confusing, but shows "Cached Credentials"
* Are the Sync/third party auth states correct? "Sign In To Sync With Cached Credentials" could emphasize that a password is still asked for when the user has cached credentials and a sync login, but should third party auth buttons be shown?

**Comment by vpomerleau** [comment]
Thanks Lauren! I've filled a follow up ticket to review the storybook states for sign in: 
https://mozilla-hub.atlassian.net/browse/FXA-10153

File: packages/fxa-settings/src/react-app-env.d.ts (hunk 1 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/react-app-env.d.ts:58]: This, plus there was an error in one of the image component names `EmailCode` vs `EmailCodeImage`

**Pull request merged**